### PR TITLE
mimic: qa/suites: Add supported-random-distro$ links.

### PR DESCRIPTION
**Pull request by yuriw**
http://tracker.ceph.com/issues/24706

### CODE REVIEW

**Comment by yuriw** [comment]
test this please

**Comment by batrick** [comment]
@yuriw you're not supposed to add your sign-off to a backport.

File: qa/suites/fs/upgrade/supported-random-distros$ [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../distros/supported-random-distro$/
```
[the remark this answers is not in the document]
Comment (yuriw) [reply qa/suites/fs/upgrade/supported-random-distros$]: can't remember details, but fixed
@batrick

**Comment by batrick** [comment]
@yuriw let's wait for the usual backport testing before merging htis. Thanks for the quick update.

**Comment by batrick** [comment]
Also, @yuriw backport PR body comment should only contain backport tracker ticket. I've fixed it again! :P

**Comment by yuriw** [comment]
[wip-yuri4-testing-2018-07-06-1935-mimic](https://trello.com/c/V9NrYNH4/338-wip-yuri4-testing-2018-07-06-1935-mimic)

**Pull request merged**